packages/brokeneck-react/src/components/Admin.test.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react'
```

Comment:
I would test this component in a different way, because you're mixing too many things together here. The Admin component is the entry component and by testing it in this way you're testing the whole application.

Now, this is not necessarily _wrong_, but what we're would usually aim for is unit tests, meaning to test a unit in isolation. Here you are testing everything that the Admin component does, including the behavior of its children.

Instead, I would aim at testing only what the Admin component itself does, meaning:

- loading the root data
- rendering the navigation
- the routing rules

You can do that by mocking the components used by the Admin component, so that you are not testing their behavior while testing Admin.

Then, the behavior of each of the components rendered by Admin can be tested in isolation.